js/sider/OppfolgingsplanerSide.js

```diff
@@ -248,6 +279,7 @@ const OppfolgingsdialogerContainer = connect(mapStateToProps, {
   hentPerson,
   hentKontaktinfo,
   hentSykmeldt,
+  hentDineSykmeldteMedSykmeldinger: hentDineSykmeldteMedSykmeldinger,
```

Comment:
Tror ikke du trenger å name variabel dersom den har identisk navn, så kan skrive hentDineSykmeldteMedSykmeldinger direkte i stedet for hentDineSykmeldteMedSykmeldinger: hentDineSykmeldteMedSykmeldinger

...samme med dineSykmeldteMedSykmeldinger over